health-sdk/sdk/src/main/res/values-en/strings.xml

```diff
@@ -43,10 +43,12 @@
     <string name="ghs_faq_answer_2">Yes, transferring the payment data to the selected banking app is free of charge. Charges may apply for the actual transfer, depending on the account model - please contact your bank for further details.</string>
     <string name="ghs_faq_answer_3">Yes, the data is transferred to the banking app in encrypted form via a Gini server. Gini receives the payment data and forwards it to the banking app. There you always have the option of checking the payment data before the transfer is executed. Gini has concluded contracts with both the insurance company and the banks for this purpose, and we are regularly audited by both.</string>
     <string name="ghs_faq_answer_4">Gini makes simplifies payments. The Munich-based company behind the photo payment works with the leading German banks and insurance companies to enable direct payment with your bank.
-Gini is ISO 27001 certified for maximum data security and operates its own server machines in an ISO 27001 certified data center in Germany. Further information can be found in the privacy policy[LINK] and on the %s website.</string>
+Gini is ISO 27001 certified for maximum data security and operates its own server machines in an ISO 27001 certified data center in Germany. Further information can be found in the %p and on the %s website.</string>
     <string name="ghs_faq_answer_5">Whether a photo of an invoice, screenshot or digital PDF - any format is suitable. Please just make sure that all payment information such as IBAN, recipient, purpose and amount are visible and not cut off.</string>
     <string name="ghs_faq_answer_6">The banks that support the Gini payment function are displayed in the bank selection menu. To use it, you must have installed your bank\'s mobile banking app on the same smartphone or tablet on which you use the insurance app. If you have not installed any of the banking apps, you can download them from the Apple App Store or Google Playstore. Subsequent activation of the banking app may be necessary.</string>
     <string name="ghs_gini_link">Gini</string>
     <string name="ghs_gini_link_url">https://gini.net/en</string>
     <string name="ghs_more_information_fragment_title">More Information</string>
+    <string name="ghs_privacy_policy">privacy policy</string>
+    <string name="ghs_privacy_policy_link_url"/>
```

Comment:
[Request] I discovered that by having an English localized placeholder then if clients don't provide a link for English locale the link won't be available if users switch the device language to English. Steps to reproduce:
- Override `ghs_privacy_policy_link_url` in the example app's `values/strings.xml`
- Set device language to English and run the example app -> The link won't be opened because Android will use the empty string from the SDK's `values-en`
- Set device language to to German -> The link will work because Android will use the overridden string from `values`.

To avoid forcing clients to also add the link for English we should remove it from here. Clients may still provide links for different locales if they prefer.